pages/home/Home.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Home = () => (
+  <div className="home">
+    <div className="home__header">
+      <span className="home__header-logo">Señor Tutor</span>
+      <div className="home__header-sections">
+        <span className="home__header-tag">CURSOS</span>
```

Comment:
Set the uppercase case with `text-transform: uppercase;`

```suggestion
        <span className="home__header-tag">Cursos</span>
```

Reply:
Done